### PR TITLE
feat(calendar): select past event titles

### PR DESCRIPTION
**Pull request by naitan79**
## Changes
Adding list of past events titles in the title input of the event details modal

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation according to my changes.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: web/src/components/modal-event-details/index.tsx [hunk 1]
```diff
@@ -22,6 +23,8 @@ import { useModal } from 'src/hooks/useModal';
 import { mutate } from 'swr';
 import * as yup from 'yup';
 
+const MuiAutocomplete = MuiAutocompleteSelect.default;
```
Comment (jonathan-delanoye-imec) [inline web/src/components/modal-event-details/index.tsx:26]: Why is this necessary?

File: web/src/components/modal-event-details/index.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { RepeatEvent } from '@constants';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { IModalEventDetails } from '@interfaces';
 import { TextField, Tooltip } from '@mui/material';
+import * as MuiAutocompleteSelect from '@mui/material/Autocomplete';
```
Comment (jonathan-delanoye-imec) [inline web/src/components/modal-event-details/index.tsx:12]: Can't we just import it like this? 

`import { TextField, Tooltip, MuiAutocompleteSelect } from '@mui/material';`

File: web/src/components/modal-event-details/index.tsx (hunk 1 above)
Comment (jonathan-delanoye-imec) [inline web/src/components/modal-event-details/index.tsx:26]: Why is this necessary?

**Review by jonathan-delanoye-imec — CHANGES_REQUESTED** [review]

File: web/src/components/modal-event-details/index.tsx [hunk 3]
```diff
@@ -8,7 +8,11 @@ import { InfoIcon } from '@components/icons/InfoIcon';
 import { RepeatEvent } from '@constants';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { IModalEventDetails } from '@interfaces';
-import { TextField, Tooltip } from '@mui/material';
+import {
+  Autocomplete as MuiAutocomplete,
```
Comment (jonathan-delanoye-imec) [inline web/src/components/modal-event-details/index.tsx:12]: Any reason why we're renaming this component as `MuiAutocomplete`?

**Review by jonathan-delanoye-imec — APPROVED** [review]

File: web/src/components/modal-event-details/index.tsx (hunk 3 above)
Comment (naitan79) [reply web/src/components/modal-event-details/index.tsx:12]: yes, there is already an Autocomplete:
`import Autocomplete from 'react-google-autocomplete';`

Comment (jonathan-delanoye-imec) [reply web/src/components/modal-event-details/index.tsx:12]: Oh yeah! Totally forgot about that one hehe
